tools/local-dev.sh

```diff
@@ -1,16 +1,26 @@
 #!/bin/bash
 
-# Run this script from the top-level directory "terra-cli/".
-# source tools/local-dev.sh
+## This script sets up the environment for local development.
+## Dependencies: docker, chmod
+## Usage: source tools/local-dev.sh
 
-echo "Pulling the default Docker image from GCR"
-./gradlew pullDockerImage
+## The script assumes that it is being run from the top-level directory "terra-cli/".
 
 echo "Building Java code"
-./gradlew install
+./gradlew clean install
 
 echo "Aliasing JAR file"
-alias terra=$(pwd)/build/install/terra-cli/bin/terra-cli
+alias terra=$(pwd)/build/install/terra-cli/bin/terra
 
 echo "Setting the Docker image id to the default"
-terra app set-image --default
\ No newline at end of file
+terra app set-image --default
+
+echo "Pulling the default Docker image from GCR"
+defaultDockerImage=$(terra app get-image)
+docker pull $defaultDockerImage
```

Comment:
Practically, I know it will be GCR because that's where publish-docker.sh uploads the image, but good point that this script should not care. Removed the "from GCR" part.